packages/node/src/transports/base.ts

```diff
@@ -198,7 +200,10 @@ export abstract class BaseTransport implements Transport {
   }
 
   /** JSDoc */
-  protected async _send(sentryReq: SentryRequest, originalPayload?: Event | Session): Promise<Response> {
+  protected async _send(
+    sentryReq: SentryRequest,
+    originalPayload?: Event | Session | SessionAggregates,
```

Comment:
We do not need to send `originalPayload` of type `SessionAggregates` because that is only required for rate limiting

Reply:
Typescript needs this or it will complain :/